Jenkinsfile

```diff
@@ -3,14 +3,16 @@ pipeline {
     
     environment {
         region = 'us-east-1'
+        validation_tag = sh (returnStdout: true, script: '[ "$GIT_BRANCH" = "master" ] && echo development || echo ""').trim()
```

Comment:
Is `"$GIT_BRANCH"` being set to `"master"`?

Reply:
Yes, every time AMI should be built from master branch only for it to be tagged as development